File: cos/olympus/src/main/java/cos/olympus/game/Usr.java
package cos.olympus.game;

public class Usr {

    public final int id;
    public final String name;
    public String worldName;

    public Usr(int id, String worldName) {
        this.id = id;
        this.name = "user:" + id;
        this.worldName = worldName;
    }

    @Override
    public String toString() {
        return "Usr{" +
                "id=" + id +
                ", worldName='" + worldName + '\'' +
                '}';
    }
}

File: cos/src/main/java/cos/olympus/game/Usr.kt
package cos.olympus.game

class Usr(
    val id: Int,
    var worldName: String
) {
    val name: String = "user:$id"

    override fun toString(): String {
        return "Usr{" +
            "id=" + id +
            ", worldName='" + worldName + '\'' +
            '}'
    }
}
